mobile-app/components/Board.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import Svg from 'react-native-svg';
+import PlayableCell from './PlayableCell';
+import BorderCell from './BorderCell';
+import { GameState } from '../../web-app/src/common/gameState';
+import { CellType } from "../utils";
+
+const CELL_STROKE_COLOR = "white";
+const CELL_SIZE = 20;
+const DISTANCE_BETWEEN_HEXAGONE_PARALLEL_SIDES = Math.sqrt(3);
+
+interface BoardData {
+  gameState: GameState;
+}
+
+export default function Board(props: BoardData) {
+  return (
+    <Svg width={getApproximateSvgSize(props.gameState.board.length)} height={getApproximateSvgSize(props.gameState.board.length)}>
+      {
+        generateBoardCells(props.gameState).map((cell, index) => (
+          cell.cellType == "playable" ?
+            <PlayableCell
+              key={index}
+              svgPoints={cell.svgPointsToDraw}
+              strokeColor={CELL_STROKE_COLOR}
+              cellValue={props.gameState.board[cell.withoutBorderCoordinates.y][cell.withoutBorderCoordinates.x]}
+              onCellPress={() => onCellPress(cell.withoutBorderCoordinates)}
+            />
+            :
+            <BorderCell
+              key={index}
+              svgPoints={cell.svgPointsToDraw}
+              strokeColor={CELL_STROKE_COLOR}
+              playerBorder={cell.cellType}
+            />
+        ))
+      }
+    </Svg >
+  );
+}
+
+function getApproximateSvgSize(boardSize: number) {
+  return boardSize * CELL_SIZE * 3;
+}
+
+function onCellPress(coordinates: { x: number, y: number }) {
+  alert(`x:${coordinates.x} y:${coordinates.y}`)
+}
+
+function generateBoardCells(gameState: GameState): { withoutBorderCoordinates: { x: number, y: number }, svgPointsToDraw: string, cellType: CellType }[] {
+  const cells = [];
+  const sizeWithBorder = gameState.board.length + 2;
+  const offset = (DISTANCE_BETWEEN_HEXAGONE_PARALLEL_SIDES * CELL_SIZE) / 2;

```

Comment:
offset is also a constant and should be declared just below DISTANCE_BETWEEN_HEXAGONE_PARALLEL_SIDES  or CELL_SIZE